src/components/modal/v2/styles/components/_header.scss

```
@@ -264,6 +264,12 @@
                     }
                 }
             }
+            &.headline-gb-preapproved {
+                max-width: 84%;
+                @include smallMobile {
+                    max-width: 81%;
+                }
+            }
```

```suggestion
            &.headline-gb-preapproved {
                max-width: 50%;           
                
                @include mobile {
                    max-width: 75%;
                }
            }
            
            &.headline-us-preapproved {
                @include smallMobile {
                    max-width: 75%;
                }
            }
```
Just some small tweaks based on some testing.